src/components/pages/Offers/OfferItem/OfferItem.jsx

```diff
@@ -8,8 +8,20 @@ import { NavLink } from 'react-router-dom'
 
 import Price from '../../../layout/Price'
 import Thumb from '../../../layout/Thumb'
+import OfferPreviewLink from '../../../layout/OfferPreviewLink/OfferPreviewLink'
+import { buildWebappDiscoveryUrl } from '../../../layout/OfferPreviewLink/buildWebappDiscoveryUrl'
 
 class OfferItem extends Component {
+  handleHrefClick = () => event => {
```

Comment:
Je pense que si tu remplaces `onClick={this.handleHrefClick()}` par `onClick={this.handleHrefClick}`, ça te permettrait de supprimer la fonction intermédiaire.

```suggestion
  handleHrefClick = event => {
```

Reply:
👍